### PR TITLE
OKTA-406710 Update domain API docs

### PR DESCRIPTION
**Pull request by anitadongare-okta**
## Description:

- Update docs for Domains APIs
- Add two new fields (`certificateSourcetype` , `expiration`) in the response object
- Monolith release: 2021.07.0

### Resolves:

* [OKTA-406710](https://oktainc.atlassian.net/browse/OKTA-406710)

### CODE REVIEW

**Review by albertchen-okta — APPROVED** [review]
lgtm thanks!

**Review by DanMacDonald-okta — APPROVED** [review]
🚢

**Comment by albertchen-okta** [comment]
Thanks Anita, we can release this when 2021.07.0 goes out to preview on July 8 (next Thurs)

File: packages/@okta/vuepress-site/docs/reference/api/domains/index.md [hunk 1]
```diff
@@ -576,26 +570,21 @@ The DomainResponse object defines the following properties:
 
 ### DomainListResponse object
 
-#### DomainListResponse properties
-
-The DomainListResponse object defines several properties:
-
+The DomainListResponse object defines list of domains, with a subset of the properties for each domain:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/domains/index.md:573]:
```suggestion
The DomainListResponse object defines list of domains with a subset of the properties for each domain:
```

File: packages/@okta/vuepress-site/docs/reference/api/domains/index.md [hunk 2]
```diff
@@ -576,26 +570,21 @@ The DomainResponse object defines the following properties:
...
 | ------------------------- | -------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| `domain`                  | String                                                         | Domain name                                                                               |
-| `id`                      | String                                                         | Domain ID                                                                                 |
-| `_links`                  | [Links](#links-object)                                         | Link relations for this object                                                            |
-| `validationStatus`        | String                                                         | Status of the Domain. Accepted values: `NOT_STARTED`, `IN_PROGRESS`, `VERIFIED`, `COMPLETED` |
-
+| `domains`                  | Array of [DomainResponse](#domainresponse-object)             | Describes zero or more domains       |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/domains/index.md:577]:
```suggestion
| `domains`                  | Array of [DomainResponse](#domainresponse-object)             | Describes the domains       |
```

**Review by susanharper-okta — COMMENTED** [review]
nits

**Pull request merged**